src/ignitus-Dashboard/Styles/style.scss

```diff
@@ -0,0 +1,28 @@
+// import ignitus token variables
+@import './src/ignitus-Shared/styles/ignitus-tokens';
+
+.banner{
+  margin-top: 12vh;
+  margin-bottom: 10px;
+  padding-top: 10px;
+  width: 100%;
+  background-color: $ignite-color-neutral-97;
+}
+.maintenanceHeading{
+  margin: 0;
+  font-family: Raleway;
```

Comment:
Font family is already there in a shared CSS file, so no need to use it again and again